lib/schema/boolean.js

```diff
@@ -55,12 +55,21 @@ SchemaBoolean.prototype.checkRequired = function(value) {
  * @api private
  */
 
-SchemaBoolean.prototype.cast = function(value) {
+SchemaBoolean.prototype.cast = function(value, model) {
   if (value === null) {
     return value;
   }
 
-  if (!this.options.strictBool) {
+  if (this.options.strictBool || (model.schema.options.strictBool && this.options.strictBool !== false)) {
```

Comment:
`model` might not be defined if we're casting for a query, which is why the tests are failing on travis. Good work thus far but to get this merged we need to figure out what to do with that case so the tests succeed.